packages/fhir-location-management/src/components/LocationForm/index.tsx

```diff
@@ -110,8 +111,8 @@ const LocationForm = (props: LocationFormProps) => {
 
   const [form] = Form.useForm();
 
-  React.useEffect(() => {
-    form.setFieldsValue({ ...initialValues });
+  useDeepCompareEffect(() => {
+    form.resetFields();
```

Comment:
https://github.com/opensrp/web/pull/974/commits/a0552f0a67f811f6d465c029eb5e01649e2654e8

Reply:
@p-netm I still feel we should show a loader as the list is re-evaluated

Reply:
@machariamuguku this will show a banner when the location hierarchy is being refetched: https://github.com/opensrp/web/pull/974/commits/ade8d650a5c8c63bb1640a97d5d4c20cbf17b46a